backend/python/app/tests/services/test_story_service.py

```diff
@@ -136,3 +143,47 @@ def test_update_story_translation_content_invalid_content_id():
 
 def test_get_story_translations_available_for_review():
     pass
+
+
+def test_remove_reviewer_from_story_translation(app, db, services):
+    story_translation = create_story_translation(db)
+    assert StoryTranslation.query.get(story_translation.id).reviewer_id != None
+    services["story"].remove_user_from_story_translation(
+        story_translation_id=story_translation.id, user_id=story_translation.reviewer_id
+    )
+    assert StoryTranslation.query.get(story_translation.id).reviewer_id == None
+
+
+def test_remove_translator_from_story_translation(app, db, services):
+    story_translation = create_story_translation(db)
+    assert StoryTranslation.query.get(story_translation.id).translator_id != None
+    services["story"].remove_user_from_story_translation(
+        story_translation_id=story_translation.id,
+        user_id=story_translation.translator_id,
+    )
+    assert StoryTranslation.query.get(story_translation.id) == None
+    assert StoryTranslationAll.query.get(story_translation.id).is_deleted == True
+
+
+def test_remove_user_from_invalid_story_translation(app, db, services):
+    st_id = 31415
```

Comment:
nit: just to be safe, a negative id would be stronger in this test as mysql ids are always positive